docs/usage/models.md

```diff
@@ -152,6 +152,14 @@ _(This script is complete, it should run "as is")_
     The example above works because aliases have priority over field names for
     field population. Accessing `SQLModel`'s `metadata` attribute would lead to a `ValidationError`.
 
+You can also achieve the same thing by passing keyword arguments to `from_orm`, instead of
+using Field aliases:
+
+```py
+{!.tmp_examples/models_orm_mode_kwargs.py!}
+```
+_(This script is complete, it should run "as is")_
```

Comment:
@dosisod, this should fix docs build:

````suggestion
{!.tmp_examples/models_orm_mode_kwargs.md!}
````

You can test this locally as well, but you will need to pull changes from remote.